exercises/poker/.meta/src/reference/java/Hand.java

```diff
@@ -2,108 +2,129 @@
 import java.util.stream.Collectors;
 
 class Hand {
```

Comment:
It's quite difficult to see what changes you've made in this file since you've changed the indentation. Please change the indentation back to the way it was (indented with 4 spaces). It's good to keep indentation consistent because that should make code easier to read and maintain 🙂